.github/workflows/build-mbed-cli-1.yaml

```diff
@@ -0,0 +1,26 @@
+name: Mbed CLI 1
```

Comment:
@pamolloy I think we should not build with mbed-cli. Why can't we just use 'make' commands to build the projects as make script in no-os already supports mbed platform?